front-end-components/pipelines/front-end.yaml

```diff
@@ -1,10 +1,11 @@
 pool:
-  vmImage: 'windows-2022'
+  vmImage: 'ubuntu-latest'
```

Comment:
This halves the build time for the front end project

Reply:
WTF

Reply:
Yeah, lets have a look once you've got the front-end sorted